values/binary.go

```diff
@@ -76,7 +76,8 @@ var binaryFuncLookup = map[BinaryFuncSignature]BinaryFunction{
 	{Operator: ast.AdditionOperator, Left: semantic.Duration, Right: semantic.Duration}: func(lv, rv Value) (Value, error) {
 		l := lv.Duration()
 		r := rv.Duration()
-		return NewDuration(l + r), nil
+		d := ConvertDuration(l.Duration() + r.Duration())
```

Comment:
Yea it would. But, until we merge that, I don't want to break this. I also didn't want to add a new method for something that's potentially getting removed.